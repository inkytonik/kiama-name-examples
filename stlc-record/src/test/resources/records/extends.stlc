{a=1, c=2} extends {a={}, b=3}
