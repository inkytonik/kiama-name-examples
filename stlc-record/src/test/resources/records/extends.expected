src/test/resources/records/extends.stlc:1:1:info: {a = 1, c  ...: {a : num, c : num, b : num}
{a=1, c=2} extends {a={}, b=3}
^
src/test/resources/records/extends.stlc:1:1:info: {a = 1, c  ...: {a : num, c : num}
{a=1, c=2} extends {a={}, b=3}
^
src/test/resources/records/extends.stlc:1:2:info: a def: field in value num
{a=1, c=2} extends {a={}, b=3}
 ^
src/test/resources/records/extends.stlc:1:4:info: 1: num
{a=1, c=2} extends {a={}, b=3}
   ^
src/test/resources/records/extends.stlc:1:7:info: c def: field in value num
{a=1, c=2} extends {a={}, b=3}
      ^
src/test/resources/records/extends.stlc:1:9:info: 2: num
{a=1, c=2} extends {a={}, b=3}
        ^
src/test/resources/records/extends.stlc:1:20:info: {a = {}, b ...: {a : {}, b : num}
{a=1, c=2} extends {a={}, b=3}
                   ^
src/test/resources/records/extends.stlc:1:21:info: a def: field in value {}
{a=1, c=2} extends {a={}, b=3}
                    ^
src/test/resources/records/extends.stlc:1:23:info: {}: {}
{a=1, c=2} extends {a={}, b=3}
                      ^
src/test/resources/records/extends.stlc:1:27:info: b def: field in value num
{a=1, c=2} extends {a={}, b=3}
                          ^
src/test/resources/records/extends.stlc:1:29:info: 3: num
{a=1, c=2} extends {a={}, b=3}
                            ^
{a : num, c : num, b : num}
